scripts/app

```diff
@@ -37,8 +37,15 @@ list_installed_apps() {
 
 # Deterministically derives 128 bits of cryptographically secure entropy
 derive_entropy () {
+  # Make sure we use the seed from the real Umbrel installation if this is
+  # an OTA update.
+  SEED_FILE="${UMBREL_ROOT}/db/umbrel-seed/seed"
+  if [[ -f "${UMBREL_ROOT}/../.umbrel" ]]; then
```

Comment:
Are you able to just add a sanity check the the current `SEED_FILE` is missing here too.

Reply:
```suggestion
  if [[ ! -f "${SEED_FILE}" ]] && [[ -f "${UMBREL_ROOT}/../.umbrel" ]]; then
```